scripts/generate-types.ts

```diff
@@ -196,7 +182,7 @@ const run = () => {
     '"*",',
     '"error",',
     ...properties.map(([key]) => `"${key}",`),
-    "];",
+    "] as const;",
```

Comment:
```suggestion
    "]",
```

This doesn't need to be `const`.

Reply:
Otherwise `typeof emitterEventNames[number]` is `string` vs. `"push" | "error" | "public" | "label" | "meta" | "fork" | "ping" | "repository" | "milestone" | "member" | "team" | "status" | "installation" | "project" | "deployment" | "release" | ... 189 more ... | "workflow_run.requested"`

Reply:
sorry that's my bad - I thought you were `const`ing the array.